Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM golang:1.5.3-onbuild
+FROM golang:1.5.4-alpine
 
-# update debian packages
-RUN apt-get update
-
-# install uchiwa-web bower package
-RUN apt-get install -yq nodejs npm git wget
-RUN ln -s /usr/bin/nodejs /usr/bin/node
-RUN npm install --production --unsafe-perm
+# golang alpine doesn't have ONBUILD, do it manually, then run npm and cleanup
+COPY . /go/src/app
+WORKDIR /go/src/app
+RUN apk add --no-cache nodejs git && \
+    go get -d -v && \
```

Comment:
The go source files can be deleted after build as well. I recommend using [glide](http://glide.sh/) or some other vendoring tool, which installs the deps into a single `vendor/` directory which can be easily removed after build.